### PR TITLE
Create DB-5.0.26-GCC-4.9.2.eb

### PR DESCRIPTION
**Pull request by levlafayette**

### CODE REVIEW

File: easybuild/easyconfigs/d/DB/DB-5.0.26-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+version = '5.0.26'
+
+homepage = 'http://www.oracle.com/technetwork/products/berkeleydb'
+
+description = """
+ Berkeley DB enables the development of custom data management solutions,
+ without the overhead traditionally associated with such custom projects.
+"""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/d/DB/DB-5.0.26-GCC-4.9.2.eb]:
```suggestion
toolchain = {'name': 'GCCcore', 'version': '4.9.2'}
```

File: easybuild/easyconfigs/d/DB/DB-5.0.26-GCC-4.9.2.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# download via http://www.oracle.com/technetwork/products/berkeleydb/downloads/,
+# requires registration
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('binutils', '2.28'),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/d/DB/DB-5.0.26-GCC-4.9.2.eb]:
```suggestion
    ('binutils', '2.25'),
```

File: easybuild/easyconfigs/d/DB/DB-5.0.26-GCC-4.9.2.eb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ Berkeley DB enables the development of custom data management solutions,
+ without the overhead traditionally associated with such custom projects.
+"""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+# download via http://www.oracle.com/technetwork/products/berkeleydb/downloads/,
+# requires registration
+sources = [SOURCELOWER_TAR_GZ]
+
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/d/DB/DB-5.0.26-GCC-4.9.2.eb]: missing checksum

**Comment by boegel** [comment]
No updates since Sept 2019, so I'll assume this is no longer relevant and close this PR

**Pull request closed without merging**